yt_dlp/extractor/telemundo.py

```diff
@@ -0,0 +1,60 @@
+# coding=utf-8
+from __future__ import unicode_literals
+
+import json
+
+from .common import InfoExtractor
+from ..utils import (
+    try_get,
+    HEADRequest,
+    unified_timestamp
+)
+
+
+class TelemundoIE(InfoExtractor):
+
+    _VALID_URL = r'https?:\/\/(?:www\.)?telemundo\.com\/.+?(?=video)video\/[^\/]+(?P<id>tmvo\d{7})'
+    _TESTS = [{
+        'url': 'https://www.telemundo.com/noticias/noticias-telemundo-en-la-noche/empleo/video/esta-aplicacion-gratuita-esta-ayudando-los-latinos-encontrar-trabajo-en-estados-unidos-tmvo9829325',
+        'info_dict': {
+            'id': 'tmvo9829325',
+            'timestamp': 1621396800,
+            'title': 'Esta aplicación gratuita está ayudando a los latinos a encontrar trabajo en Estados Unidos',
+            'uploader': 'Telemundo',
+            'uploader_id': 'NBCU_Telemundo',
+            'ext': 'mp4',
+            'upload_date': '20210519',
+        },
+        'params': {
+            'skip_download': True,
+        }
+    }, {
+        'url': 'https://www.telemundo.com/shows/al-rojo-vivo/empleo/video/personajes-de-times-square-piden-que-la-ciudad-de-nueva-york-los-deje-volver-trabajar-tmvo9816272',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        metadata = json.loads(
+            self._search_regex(r'<[^>]+id="__NEXT_DATA__"[^>]+>([^<]+)', webpage, 'JSON metadata'))
+        redirect_url = try_get(
+            metadata,
+            lambda x: x['props']['initialState']['video']['associatedPlaylists'][0]['videos'][0]['videoAssets'][0]['publicUrl'])
+
+        m3u8_url = self._request_webpage(HEADRequest(
+            redirect_url + '?format=redirect&manifest=m3u&format=redirect&Tracking=true&Embedded=true&formats=MPEG4'),
+            video_id, 'Processing m3u8').geturl()
```

Comment:
is this needed?

Reply:
Yep, gets the Akamai account. If that changes, we wouldn't know from the original URL.

Reply:
ok, gotcha